### PR TITLE
MNT: Suppress pytest warning

### LINKED ISSUE #3: PytestAssertRewriteWarning: Module already imported so cannot be rewritten
This warning was seen in `astropy` (xref astropy/astropy#9214) but @bsipocz suggested that I open issue here:
```
/home/travis/miniconda/envs/test/lib/python3.7/site-packages/_pytest/config/__init__.py:833
  /home/travis/miniconda/envs/test/lib/python3.7/site-packages/_pytest/config/__init__.py:833:
PytestAssertRewriteWarning: Module already imported so cannot be rewritten:
pytest_astropy_header
    self._mark_plugins_for_rewrite(hook)
```

@jdavies-st attempted to address it at astropy/astropy#9437 but it was not merged.

### PR DESCRIPTION
**Pull request by pllim**
Port over astropy/astropy#9437 by @jdavies-st, as suggested by @bsipocz .

Tests will probably fail because #2 is unresolved.

Fixes #3

### CODE REVIEW

**Comment by astrofrog** [comment]
@pllim - can you rebase and also add a couple of newlines before ``PYTEST_DONT_REWRITE`` (since it doesn't matter where PYTEST_DONT_REWRITE is in the docstring as long as it's there)

**Comment by pllim** [comment]
@astrofrog , done.

File: setup.cfg [hunk 1]
```diff
@@ -32,3 +32,6 @@ test =
 [options.entry_points]
 pytest11 =
     pytest_astropy_header = pytest_astropy_header.display
+
+[flake8]
+max-line-length = 125
```
Comment (bsipocz) [inline setup.cfg:37]: how come this number? 

(it's OK, I just wonder why it's not 80 or 100 or 120, 125 is honestly just as good, just a bit unusual)

Comment (pllim) [reply setup.cfg:37]: I can try 120 but pretty sure my Emacs would still complain if it is 100.

Comment (pllim) [reply setup.cfg:37]: Answer: It was arbitrarily chosen to silence `flake8`.

Comment (bsipocz) [reply setup.cfg:37]: 125 it is then. I really just wondered 😄

**Review by bsipocz — APPROVED** [review]

File: setup.cfg (hunk 1 above)
Comment (pllim) [reply setup.cfg:37]: If you like power of 2, I can increase to 128. Or even 129 if you want 128 *without* the null terminator. :smirk:

**Comment by bsipocz** [comment]
Would you merge this @astrofrog? I don't have the power.

**Review by astrofrog — APPROVED** [review]
I'm so sorry it took so long for me to merge! We need to review repo permissions...

**Pull request merged**